orchestration/hca_manage/check.py

```diff
@@ -20,8 +20,7 @@ def run(arguments: Optional[list[str]] = None) -> None:
     parser.add_argument("-e", "--env", help="The Jade environment to target",
                         choices=["dev", "prod", "real_prod"], required=True)
 
-    parser.add_argument("-p", "--project", help="The Jade project to target")
-    parser.add_argument("-d", "--dataset", help="The Jade dataset to target", required=True)
+    parser.add_argument("-i", "--dataset-id", help="The Jade dataset ID to target", required=True)
```

Comment:
This is unrelated; passing the project and dataset name is unnecessary. Since we'll always have the dataset ID, we can discover the name and google project from that by querying jade.